packages/fluent-theme/src/components/sendBox/SendBox.tsx

```diff
@@ -138,14 +139,9 @@ function SendBox(props: Props) {
         )
       );
 
-      setAttachments(newAttachments);
-
-      // TODO: Currently in the UX, we have no way to remove attachments.
-      //       Keep concatenating doesn't make sense in current UX.
-      //       When end-user can remove attachment, we should enable the code again.
-      // setAttachments(attachments => attachments.concat(newAttachments));
+      setAttachments(attachmentsRef.current.concat(newAttachments));
```

Comment:
TIL.

Reply:
The `setAttachments` doesn't accept a setter function anymore. Hope this won't introduce any stale results, not in my testing at least.